### ISSUE TITLE
Delete Page images when delete page

### ISSUE THREAD

**Issue by dimobelov** [bludit/bludit#869]
Now each page uploads pictures to own directory. They are only available for this page but when we delete page pictures stay on server, which is not good.

Here I offer a simple delete images directory code. After this line:
https://github.com/bludit/bludit/blob/0e80b211c744c93bc7be98422b8ebcf1eb3d690a/bl-kernel/pages.class.php#L282
```
		// Delete Page images directory
		$dir_uuid = $this->db[$key]["uuid"];
		if(!empty($dir_uuid)){			
			Filesystem::deleteRecursive(PATH_UPLOADS_PAGES.$dir_uuid);			
		}
```

**Comment by dignajar** [comment]
I upload the fix, just call the function deleteRecursive.
Thanks for report it.

**Comment by dignajar** [comment]
Implemented in Bludit v3.5, thanks for report it!.
